src/main/java/teammates/ui/output/FeedbackSessionData.java

```diff
@@ -89,12 +94,12 @@ public FeedbackSessionData(FeedbackSessionAttributes feedbackSessionAttributes)
         if (feedbackSessionAttributes.isOpened()) {
             this.submissionStatus = FeedbackSessionSubmissionStatus.OPEN;
         }
-        if (feedbackSessionAttributes.isClosed()) {
-            this.submissionStatus = FeedbackSessionSubmissionStatus.CLOSED;
-        }
         if (feedbackSessionAttributes.isInGracePeriod()) {
             this.submissionStatus = FeedbackSessionSubmissionStatus.GRACE_PERIOD;
         }
+        if (feedbackSessionAttributes.isClosed()) {
+            this.submissionStatus = FeedbackSessionSubmissionStatus.CLOSED;
+        }
```

Comment:
Referencing the related review: https://github.com/TEAMMATES/teammates/pull/11606#discussion_r821801336

Reply:
I initially removed the whole thing (so I could no longer see the original order) and wrote other stuff with the new order, but I put it back when refactoring but still with the new order, so now it merely looks like a reordering. There is a [more detailed explanation](https://github.com/TEAMMATES/teammates/pull/11606#discussion_r821902160), but the important part is we kept it because of the rationale I had when I wrote it with the new ordering.

The rationale can maybe best be explained through an example. Let's say the session is closed after (not at) 12pm. Let's say sometime right before or exactly at 12pm, in the original order, it first checks if the session is closed then it returns false. It takes time to do this, so when it later checks if the session is in the grace period, it may already be after 12pm, so it may also return false here. Then `submissionStatus` is never set, which I think is probably not good, right? However, in the new order, right before or at 12pm, it first checks if the session is in the grace period then it returns true and `submissionStatus` is set. Sure, by the time the response is sent back, it would be after 12pm, so the `submissionStatus` would be wrong already, but I think this is still better than being left unset.

I can still change it back to the original order though, if you'd like, because I know it's not within the scope of the PR, or if you think my rationale is invalid.